.github/workflows/performance.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           name: kafka producer perf test Benchmark
           tool: 'customSmallerIsBetter'
           output-file-path: output.json
-          gh-pages-branch: gh-pages
+          gh-pages-branch: performance-results
```

Comment:
Does this mean we don't have an automatic page displaying the results on the website?

Reply:
correct, though currently the results are also currently not visible in our `gh-pages` branch [by choice](https://github.com/kroxylicious/kroxylicious/issues/177#issuecomment-1455863466). Can open a separate issue for publishing benchmark results somewhere in future.